Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -188,6 +203,7 @@ theorem IsInvariantBlock.isFixedBlock {B : Set X} (hfB : IsInvariantBlock G B) :
 theorem IsInvariantBlock.isBlock {B : Set X} (hfB : IsInvariantBlock G B) :
     IsBlock G B :=
   hfB.isFixedBlock.isBlock
+@[deprecated (since := "2024-09-14")] alias isFixedBlock_top := IsInvariantBlock.isBlock
```

Comment:
Do you mean to say `alias isFixedBlock_top := isFixedBlock_univ`?

Reply:
Yes, and it was not at the right place.